packages/relay/tests/lib/eth.spec.ts

```diff
@@ -651,7 +670,7 @@ describe('Eth calls using MirrorNode', async function () {
   });
 
 
-  it('"eth_blockNumber" should return the latest block number using cache', async function () {
+  it.only('"eth_blockNumber" should return the latest block number using cache', async function () {
```

Comment:
Why this only?

Reply:
Ugh!  I removed it but didn't stage the change.  Thanks for catching this.